react/tic-tac-toe/src/app/screens/Game/components/MatchHistory/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import styles from './styles.module.scss';
+import matches from "~services/MatchesService";
+import { connect } from 'react-redux';
+import matchActions from "~redux/matches/actions";
+import Spinner from 'react-spinkit';
+import { arrayOf, func, boolean } from 'prop-types';
+import { matchPropType } from "~constants/propTypes.js"
+import {PLAYER_ONE, PLAYER_TWO, WINNER, MATCH_HISTORY_TITLE, GET_ERROR} from "~constants";
+
+class MatchHistory extends Component {
+
+    componentDidMount = () => {
+      const { toggleLoading, getMatches} = this.props;
+      toggleLoading();
+      matches.getMatches().then(response => getMatches(response.data));
```

Comment:
we do service calls, and response handling in the redux's action, here it doesnt make much sense to send the response to redux and then get it again from redux 😛

Reply:
I talked this with erica, the problem was that at this time of the training i didn't had redux thunk, so i wasn't able to do service call in redux action(because they needed to be pure), i fixed this in next PR.